src/briefcase/integrations/java.py

```diff
@@ -310,8 +310,19 @@ def install(self):
 
     def uninstall(self):
         """Uninstall a JDK."""
-        with self.tools.console.wait_bar("Removing old JDK install..."):
-            if self.tools.host_os == "Darwin":
-                self.tools.shutil.rmtree(self.java_home.parent.parent)
-            else:
-                self.tools.shutil.rmtree(self.java_home)
+        try:
+            with self.tools.console.wait_bar("Removing old JDK install..."):
+                if self.tools.host_os == "Darwin":
+                    jdk_location = self.java_home.parent.parent
+                else:
+                    jdk_location = self.java_home
+                self.tools.shutil.rmtree(jdk_location)
+
+        except PermissionError as e:
+            raise BriefcaseCommandError(
+                f"""\
+Permission denied when trying to remove {jdk_location}.
```

Comment:
A really minor UX thing - but a common usage pattern if this problem occurs will be to copy the path out of the terminal output and look there; the trailing `.` can be confusing as to whether it's a part of the path or not. In cases like this, it's better to either use "bad English" and omit the period, or use quotes around the string.
```suggestion
Permission denied when trying to remove {jdk_location!r}.
```